crates/agentgateway/src/telemetry/metrics.rs

```diff
@@ -169,6 +174,37 @@ impl Metrics {
 			gen_ai_request_duration,
 			gen_ai_time_per_output_token,
 			gen_ai_time_to_first_token,
+
+			response_bytes: {
+				let m = Family::<HTTPLabels, _>::default();
+				registry.register_with_unit(
+					"response_bytes",
+					"Total HTTP response bytes sent",
+					Unit::Bytes,
+					m.clone(),
+				);
+				m
+			},
+			tcp_rx_bytes: {
+				let m = Family::<TCPLabels, _>::default();
+				registry.register_with_unit(
+					"tcp_rx_bytes",
```

Comment:
The unit gets auto-appended, so this is `agentgateway_tcp_rx_bytes_bytes_total` which is not right. Should just have

```suggestion
					"tcp_rx",
```

That being said, wondering about the metric name. Its a bit ambiguous right now.

WDYT of `agentgateway_downstream_received_bytes_total`? this mirrors  `agentgateway_downstream_connections_total` as well as istio's `tcp_received_bytes`.

Specifically I don't like `tx`/`rx`, and to some extent the lack of 'downstream'

Reply:
+1 to `agentgateway_downstream_received_bytes_total`